subcommittees/mentoring/minutes-2015-03-17.md

```diff
@@ -0,0 +1,208 @@
+-   The biggest question here (IMO) is actually not so much us seeing how a new instructor teaching, but a new instructor seeing, from an instructor´s point of view how to prepare and teach.
+
+RS: This will be one topic on half day.
+
+KL: Teach one topic for colegues with the help of another instruction.
+
+BM: Oversight?
+
+KL: First sound a little strange. Run a small session for colegues is a good introduction/gentil other than go to a full class.
+
+BM: Similar to 5 minutes lesson done in the past?
+
+RS: At the end of instructor training?
+
+BM: Yes.
+
+RS: We need a proposal.
+
+BM: I can do it.
+
+BM: What about all new instructor that didn't have another instructor nearby?
+
+RS: Skype/Google Hangout/WebRTC/...
+
+KL: We have a lack of available places to teach. Concern that this can create problems of where we can teach. Show the material, show the teaching style.  Labs can thought that after the instructor training anyone can teach.
+
+### First Workshop Helping Session
+
+BM: I ran sessions like that and had zero attended. How make it popular?
+
+KL: See instructors upscaling.
+
+BM: Lets try your way.
+
+TT: Can run it on April
+
+BM: Rotate?
+
+RS: I'm OK.
+
+KL: Two show ups.
+
+DC: can we as  members of the mentoring committee attach ourselves to the end of instructor training, and have them teach to us for about an hour?
+
+RS: will ask Greg.
+
+KL: would love to go through the training again, lots have changed since my time.
+
+### Screen record of how set up workshop page
+
+KL: Check with Greg first, I think it is fairly up to date.
+
+TT: I can do it.
+
+### Study new instructor experience
+
+BM: Period between end of Instructor Training & start of first workshop
+
+Questions:
+
+-   How prepared do new instructors feel to launch their first workshop?
+    -   Not very prepared. +1
+-   What roadblocks do people experience in this period?
+    -   Creating the workshop page.<-- address with screen recording suggestion
+        above
+    -   Know our lesson material.<-- made some progress on this in Melbourne with hard requirement of reading one unit before coming to Instructor Training, and requiring all exercises be done on that unit.
+        -   Should work on helping with this - right now, the material is usually way too much to teach in their slotted time, we should help people winnow the material into something that is more feasible to teach in the given time period.
+-   What resources will help smooth this path out?
+
+BM: How much our instructors are prepared? Students like the instructor training but they don't know what to do on the first workshop. Can we measure what blocks new instructors?
+
+KL: The python module is to be cover in 4 hours and I need 6 hours. Was frustrated for new instructors to not cover all the material and this is also bad for students.
+
+BM: The Python is two full day if cover from cover to cover.  Let instructors know that they didn't need to finished the lesson we wrote.
```

Comment:
“cover from cover to cover” → “covered from cover to cover” (or maybe “covered in its entirety”).